### PR TITLE
Add dependabot

### PR DESCRIPTION
**Pull request by wrt54g**
**What I did**
I added dependabot for actions
**Related issue**

**(not mandatory) A picture of a cute animal, if possible in relation with what you did**
image

### CODE REVIEW

**Comment by mat007** [comment]
We already have dependabot on this repo.

**Comment by wrt54g** [comment]
@mat007 Not for actions yet.

**Comment by mat007** [comment]
Ah I see what you mean, the version update was never enabled.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: weekly
+      time: "06:00"
+    open-pull-requests-limit: 10
```
Comment (mat007) [inline .github/dependabot.yml:8]: Can you replace with
```suggestion
# To get started with Dependabot version updates, you'll need to specify which
# package ecosystems to update and where the package manifests are located.
# Please see the documentation for all configuration options:
# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates

version: 2
updates:
  - package-ecosystem: "" # See documentation for possible values
    directory: "/" # Location of package manifests
    schedule:
      interval: "weekly"
```
which is the default when enabling this through the Security tab of the Settings?

**Review by mat007 — COMMENTED** [review]
You need to sign your commit, see the [DCO failure](https://github.com/docker/compose-cli/pull/2170/checks?check_run_id=7202498051).

**Comment by wrt54g** [comment]
@mat007 How do I do that in GitHub Web?

**Comment by mat007** [comment]
> How do I do that in GitHub Web?

I honestly don’t know. I’m not even sure it’s possible. 🤔

**Comment by wrt54g** [comment]
It probably is not. I can't find it anywhere.

**Pull request closed without merging**